src/styles/global.css

```
@@ -1445,6 +1445,101 @@ body.blog-page {
+  grid-column: 2 / 7;
+  grid-row: 2;
+  background: var(--surface);
+  padding: 48px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.og-label {
+  font-family: "Inter", system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)